client/src/helpers/APICalls/deletePhoto.ts

```diff
@@ -0,0 +1,18 @@
+import { FetchOptions } from '../../interface/FetchOptions';
+
+const deletePhoto = async (key: string | undefined | null, profileId: string) => {
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
```

Comment:
```suggestion
    method: 'DELETE',
```
Since we're deleting a resource or a part of it, the DELETE HTTP verb should be used here